src/menu/columnar_menu.rs

```diff
@@ -38,8 +38,8 @@ struct ColumnDetails {
     pub columns: u16,
     /// Column width
     pub col_width: usize,
-    /// The shortest of the strings, which the suggestions are based on
-    pub shortest_base_string: String,
+    /// The display width of the shortest string, which the suggestions are based on
+    pub shortest_base_string: usize,
```

Comment:
The naming is a little bit confusing. I think we're safe to change it in this private struct?